mkdocs.yml

```diff
@@ -1,7 +1,7 @@
 # yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
 site_name: DDEV Docs
 # todo: Make sure to set this to the correct one; otherwise it breaks 404 page
-site_url: https://ddev.readthedocs.io/en/latest/
+site_url: https://ddev.readthedocs.io/en/stable/
```

Comment:
Pretty weird if `site_url` is only for the 404...

Reply:
I wondered why we ever need `site_url`, and found this:

- https://github.com/mkdocs/mkdocs/issues/1598#issuecomment-1870326517

People still complain about it because it hasn't been fixed.